### ISSUE TITLE
Transferrin vs TIBC : wrong LOINC code

### ISSUE THREAD

**Issue by Mauvila** [MIT-LCP/mimic-code#593]
### Prerequisites

* [ X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.physionet.org/about/mimic/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

I believe the LOINC code for D_LABITEMS row 198 (item ID 50998, "Transferrin") is wrong.  It says 2500-7, which is the TIBC.  I believe the LOINC code should be 3034-6.  TIBC is often used instead of transferrin, but the values aren't exchangeable.  

To illustrate the issue, you can look at the SQL search:
SELECT d.LOINC_CODE, d.LABEL, e.VALUENUM AS 'Numeric value', e.* FROM labevents e, d_labitems d WHERE (e.subject_id=109) AND ((e.itemid=50953) OR (e.itemid=50998)) AND (d.itemid=e.itemid);

For a given lab draw (ie same CHARTTIME), there are two significantly different values with the same LOINC code (normalizing by the units would give you even more discrepant results).

J. Allen

**Comment by alistairewj** [comment]
I would agree, and it's odd that this LOINC is wrong as usually the lab data is pretty solid. We'll look into it on our end.

**Comment by alistairewj** [comment]
Let's keep one issue for all the bad LOINC stuff: https://github.com/MIT-LCP/mimic-code/issues/938
